src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -30,93 +36,183 @@ class DaySelector extends PureComponent<DaySelectorProps, DaySelectorState> {
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[0]}
-                            onChange={this.handleChange(0)}
-                            value="0"
-                            color="primary"
-                        />
-                    }
-                    label="Sunday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[1]}
-                            onChange={this.handleChange(1)}
-                            value="1"
-                            color="primary"
-                        />
-                    }
-                    label="Monday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[2]}
-                            onChange={this.handleChange(2)}
-                            value="2"
-                            color="primary"
-                        />
-                    }
-                    label="Tuesday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[3]}
-                            onChange={this.handleChange(3)}
-                            value="3"
-                            color="primary"
-                        />
-                    }
-                    label="Wednesday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[4]}
-                            onChange={this.handleChange(4)}
-                            value="4"
-                            color="primary"
-                        />
-                    }
-                    label="Thursday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[5]}
-                            onChange={this.handleChange(5)}
-                            value="5"
-                            color="primary"
-                        />
-                    }
-                    label="Friday"
-                />
-
-                <FormControlLabel
-                    control={
-                        <Checkbox
-                            checked={this.state.days[6]}
-                            onChange={this.handleChange(6)}
-                            value="6"
-                            color="primary"
-                        />
-                    }
-                    label="Saturday"
-                />
-            </FormGroup>
-        );
+        if (detectMobile() == false) {
```

Comment:
***Please, please*** don't do this manually. Define an array, and map through it:
https://reactjs.org/docs/lists-and-keys.html

Here's an example:
1. [Define the array of options](https://github.com/icssc/AntAlmanac/blob/aponia-monorepo/apps/client/src/components/buttons/CustomEvent.tsx#L36)
2. [Iterate through the array and generate a JSX element for each, using the value and index](https://github.com/icssc/AntAlmanac/blob/aponia-monorepo/apps/client/src/components/buttons/CustomEvent.tsx#L192)

Reply:
Agreed on this point. Also, `==false` is kinda clunky here you should rename the  function (see previous comment) and just do `!(function call)`